code/game/objects/items/devices/mental_imprinter.dm

```diff
@@ -8,7 +8,7 @@
 	var/stat_increase = 5
 	var/apply_sanity_damage = 30
 	var/spent = FALSE
-	price_tag = 1200
+	price_tag = 24
```

Comment:
these should still be expensive, as they're trade-beacon-exclusive (iirc)

Reply:
Aggreing with lolman here this is antag only, should be expensive, missed this on my first glance over it